kafka-inmemory/resources/kubernetes.yaml

```diff
@@ -136,6 +140,13 @@ metadata:
 data:
   config.yml: |
     lowercaseOutputName: true
+    rules:
+      - pattern: kafka.server<type=(.+), name=(.+)PerSec\w*><>Count
+        name: kafka_server_$1_$2_total
+      - pattern : kafka.server<type=(.+), name=(.+)PerSec\w*, topic=(.+)><>Count
+        name: kafka_server_$1_$2_total
```

Comment:
We drop the name of the topic from the `name` (I assume the `name` is the name of the metric as it is exposed to Prometheus)?

Reply:
Oh sorry I didn't notice the `labels` part. Ignore my comment.